common/app/common/TrailsToRss.scala

```diff
@@ -131,9 +131,10 @@ object TrailsToRss extends implicits.Collections {
         .filterNot(_.config.excludeFromRss)
         .flatMap(_.all)
         .filter{
-        case _: LinkSnap => false
-        case _ => true}
-        .distinctBy(_.id)
+          case _: LinkSnap => false
```

Comment:
@robertberry This is a `filter` to take the one's that aren't `LinkSnaps`. So that wouldn't work.